controllers/articlesController.js

```diff
@@ -0,0 +1,43 @@
+const { selectArticles, selectArticleById, patchArticleVotes } = require("../models/articles.model");
+
+exports.getArticles = (req, res, next) => {
+    
+    return selectArticles()
+    .then(({rows}) => {
```

Comment:
It's best practice to de-construct the rows in the model before returning the result up again into the controller.